packages/semantic-dom-diff/chai-dom-diff.d.ts

```diff
@@ -1,7 +1,24 @@
-/// <reference path="chai-dom-diff-plugin.d.ts" />
+/// <reference path="./chai-dom-diff-plugin.d.ts" />
 
 /**
- * @param {any} chai
- * @param {any} utils
+ * @param chai
+ * @param utils
  */
-export const chaiDomDiff: (chai: any, utils: any) => void;
+export const chaiDomDiff: (chai: typeof Chai, utils: any) => void;
+
+// No idea what this is
+type Snapshot = any;
```

Comment:
seriously, what is this?

Reply:
this comes from https://www.npmjs.com/package/mocha-snapshots and https://www.npmjs.com/package/karma-mocha-snapshot